hyperlinks all the links.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Hyperlinks</title>
+    </head>
+<body>
+<h1 id="top"> This page is about all the links </h1>
+<h2>Internal links  </h2>
+<section id="section1">
+     Here is how we create internal links:
+<a href="index.html" title="index html"> linking to a file in the same directory </a>
+
+<a href="index.html" title="index">
+    <div>Div linking to the iindex file</div>
+</a>
+</section>
+
+<section id="section2">
+<h2> External links</h2>
+<p>lets link to an external website 
+    <!--link to external website-->
+    <a href="http://wwww.google.com" 
+    target="_blank" title="Check out this link!">External link page</a>
+</p>
+</section>
+
+<section id="section3">
+    <h2>Links to sections of the same page</h2>
+
+        <ul>
+            <!-- Link to every section in the page-->
+            <li><a href="#section1">#section1</a></li>
+            <li><a href="#section2">#section2</a></li>
+            <li><a href="#section3">#section3</a></li>
+        </ul>
+</section>
+
+<div>
+    <h2><a name="section3>(#section3) Section 3</a></h2>
+    <p>
+        Back to top: <a href="#top"> Back to top</a>

```

Comment:
This is a working example of an anchor and its corresponding id. From line 7

id = # 

so #top links to id="top"

Reply:
yayyy okay I think i get that now